### DISCUSSION TITLE
Class based Volt components

### DISCUSSION THREAD

**Discussion by yacobii** [livewire/livewire#6382] [1. Help]
use Livewire\Volt\Component;
new class extends Component {
    public $count = 0;
    public function increment()
    {
        $this->count++;
    }
}

<div>
    <p>{{ $count }}</p>
    <button wire:click="increment">+</button>
</div>

I get an error: 
Call to undefined function Livewire\Volt\opcache_invalidate()
but without Class based Volt components it is ok.

**Comment by joshhanley** [comment]
@mehdiyaq thanks for reporting! Volt is managed by the Laravel team, so please open an issue on the Volt repo 🙂 thanks!
